src/com/lftechnology/java/training/niraj/ems/utils/Utils.java

```diff
@@ -0,0 +1,115 @@
+package com.lftechnology.java.training.niraj.ems.utils;
+
+import java.util.Map;
+
+/**
+ * Class for Utility methods
+ * 
+ * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+ * 
+ * @see StringBuilder
+ *
+ */
+public class Utils {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)